lib/msal-common/src/authority/Authority.ts

```diff
@@ -540,6 +578,15 @@ export class Authority {
                 this.hostnameAndPort
             );
         }
+
+        getCloudDiscoveryMetadataFromNetworkMeasurement?.addStaticFields({
+            httpVerCloudMetadata: response?.headers[HeaderNames.X_MS_HTTP_VERSION] || Constants.EMPTY_STRING
+        });
+
+        getCloudDiscoveryMetadataFromNetworkMeasurement?.endMeasurement({
+            success: true,
```

Comment:
I personally would prefer these to be 1 line if they're only going to have the success attribute. If there's more attributes then multi-line construction does make it more readable.